src/Revit/RevitHyparTools/RevitHyparTools.csproj

```diff
@@ -16,4 +15,7 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Elements\Elements.csproj" />
```

Comment:
No local reference as per comment above.

Reply:
same thought,  I feel like we should reference the version of Elements that is in this repo, not the nuget version.